document_page/views/document_page.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0"?>
 <openerp>
     <data>
-       <menuitem name="Knowledge"
-                 id="knowledge.menu_document" />
+      <menuitem name="Knowledge"
+                id="knowledge.menu_document" />
 
```

Comment:
No need for this. This menu item is already created here:
https://github.com/OCA/knowledge/pull/76/files#diff-8186cbed73afd58aa336582b1d4c5ae1R6